src/ui/header/mobile-navigation.tsx

```
@@ -0,0 +1,190 @@
+    >
+      <nav className="h-full overflow-y-auto" aria-label="Mobile navigation">
+        <div className="mx-auto max-w-screen-xl p-4 space-y-6">
+          <div className="flex items-center justify-between">
+            <CTAList ctas={ctas} className="grid flex-1 gap-2 *:w-full" />
+          </div>
+          <div className="h-px bg-border" role="separator" />
+          <ul className="space-y-3" role="menu">
+            {menu?.items?.map((item: MenuItem, index: number) => {
+              if (item._type === 'link') {
+                return (
+                  <li key={`mobile-${item.label}-${index}`} role="none">
```

Biome (1.9.4)

[error] Enforce that elements with ARIA roles must use a valid, non-abstract ARIA role.

Check WAI-ARIA for valid roles or provide options accordingly.
Unsafe fix: Remove the invalid role attribute.
 Check the list of all valid role attributes.


(lint/a11y/useValidAriaRole)